lms/envs/aws.py

```diff
@@ -931,6 +931,13 @@
     ENTERPRISE_CUSTOMER_LOGO_IMAGE_SIZE
 )
 
+# Course enrollment modes to be hidden in the Enterprise enrollment page
+# if the "Hide audit track" flag is enabled for an EnterpriseCustomer
+ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES = ENV_TOKENS.get(
+    'ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES',
+    ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES
+)
```

Comment:
It already is: https://github.com/edx/configuration/pull/3873/files

I think the rationale is for instances that use `honor` modes or similar as discussed on https://github.com/edx/edx-enterprise/pull/104#issuecomment-300916213